### PR TITLE
Feature: Register Payment

### PR DESCRIPTION
**Pull request by jdrieghe**
### Added
- Description for `invoices.registerPayment` endpoint
- Money data structure
- Invoicing section to full documentation 

### Removed
- Broken invoicing endpoints in `invoicing.apib` (*Note: these will be added again by another PR in a working state*)

### Fixed
- Some typos in contributing guidelines

### CODE REVIEW

File: apiary.apib [hunk 1]
```diff
@@ -949,6 +949,20 @@ Update a department
...
+## Invoices [/invoices]
+
+### invoices.registerPayment [POST /invoices.registerPayment]
+
+Add full or partial payment for an invoice
+
++ Request (application/json)
+
+    + Attributes (object)
+        + invoice_id: `83ec61a7-6e50-4ebb-a7ed-36e06938561f` (string)
```
Comment (jdrieghe) [inline apiary.apib]: maybe turning this into a uuid is going to slow us down a lot for now?

Comment (jenssegers) [inline apiary.apib]: I do think we should go for it in v2 API

File: apiary.apib [hunk 2]
```diff
@@ -949,6 +949,20 @@ Update a department
 
 + Response 204
 
+## Invoices [/invoices]
+
+### invoices.registerPayment [POST /invoices.registerPayment]
```
Comment (jenssegers) [inline apiary.apib]: Would be `invoices.pay` be too short?

File: apiary.apib (hunk 1 above)
Comment (jdrieghe) [inline apiary.apib]: While I agree with the idea, creating a simple invoice endpoint would drag us through a chain reaction:
invoiceId, subscriptionId, creditNoteId, customerId, projectId, accoundId, dealId, ledgerId, ...

Also, should we set up the "translation" service first in that case?

File: apiary.apib (hunk 2 above)
Comment (jdrieghe) [inline apiary.apib]: That could work. @Brammm @steftrenson ?

Comment (steftrenson) [inline apiary.apib]: I feel neutral about this. ;)

File: apiary.apib (hunk 1 above)
Comment (steftrenson) [inline apiary.apib]: I agree with @jenssegers here, we know we're going to do it, why not do it first instead of having to refactor it later on.

Comment (jdrieghe) [inline apiary.apib]: I read "I volunteer" @steftrenson  :)

But yeah, let's go for it, knowing that it will slow us down

File: apiary.apib (hunk 2 above)
Comment (Brammm) [inline apiary.apib]: Indifferent.

**Comment by stivni** [comment]
closing this in favor of #160

**Pull request closed without merging**